python_scripts/ensemble_sol_02.py

```diff
@@ -66,7 +66,7 @@
                            columns=data.columns)
 tree_predictions = []
 for tree in forest.estimators_:
-    tree_predictions.append(tree.predict(data_range))
+    tree_predictions.append(tree.predict(data_range.to_numpy()))
```

Comment:
```suggestion
    # we convert `data_range` into a NumPy array to avoid a warning raised in scikit-learn
    tree_predictions.append(tree.predict(data_range.to_numpy()))
```